mods/cnc/rules/defaults.yaml

```diff
@@ -293,6 +295,8 @@
 ^Helicopter:
 	Inherits@1: ^ExistsInWorld
 	Inherits@3: ^SpriteActor
+	UpdatesPlayerStatistics:
+		AddToArmyValue: true
```

Comment:
If we decided that chinooks don't count then they will need to be disabled in this mod too.